### PR TITLE
Build a single multi-arch debian-base image

### PR DESCRIPTION
**Pull request by aledbf**
**What type of PR is this?**

/kind cleanup
/kind feature

**What this PR does / why we need it**:

Use builds (already enabled) to build images, removing qemu-static-* from the process 

**Which issue(s) this PR fixes**:

Related to https://github.com/kubernetes/kubernetes/issues/87197#issuecomment-583753009

**Special notes for your reviewer**:

With this change, the docker image contains a history (right now that is empty)

**Does this PR introduce a user-facing change?**:
```release-note
Image debian-base is multi-arch, removing the need to have one image per platform.
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: hack/init-buildx.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```
Comment (BenTheElder) [inline hack/init-buildx.sh:1]: heh, this is familiar :-)

Comment (aledbf) [reply hack/init-buildx.sh:1]: on purpose :)

**Comment by BenTheElder** [comment]
I think the potential problem with this is when we build images on top of it if we can't use buildx then we have to be extra careful to select the architecture matching image (easier if they have a name each), we've recently ripped buildx out of other builds in k/k due to the release pipeline :(

**Comment by aledbf** [comment]
Closing. Thank you for the feedback @BenTheElder

**Pull request closed without merging**